ios/RNZendeskChatModule.m

```diff
@@ -30,6 +30,18 @@ @implementation RCTConvert (ZDKChatFormFieldStatus)
 
 @end
 
+@implementation RCTConvert (ZDKRating)
+
+RCT_ENUM_CONVERTER(ZDKRating
+,
+                   (@{
+                       @"good": @(ZDKRatingGood),
+                       @"bad": @(ZDKRatingBad),
```

Comment:
In theory, there are also `none` and `description` in the enum... but seems like they are not available on android, so I removed them to keep the same API